tests/molecules/C2_pp/CMakeLists.txt

```diff
@@ -1,54 +1,70 @@
-
+IF (ENABLE_SOA)
 IF (NOT QMC_CUDA)
  IF (NOT QMC_COMPLEX)
 
-# C2 TZ PP multi determinants
-# Short tests
-  LIST(APPEND C2MSDJS_VMC_SCALARS "totenergy" "-11.064964  0.000865") # total energy
-  LIST(APPEND C2MSDJS_VMC_SCALARS "variance"  "  0.149682  0.003928") # energy variance
+#C2_PP test targets the Multideterminant route of the code. The wavefunction data is  stored in a XML format AND HDF5 Format. Both files should be run
+# Energy from QP: E=-10.9015995458600 Ha
+#
+#   "kinetic" " 7.72378 0.00016") # kinetic energy
+#   "totenergy" " -11.059755 0.000013 ") # total energy
+#   "eeenergy" " 12.142072 0.000083") # e-e energy
+#   "samples" "1072889856 0") # samples
+
 
-  QMC_RUN_AND_CHECK(short-C2_pp-msdj_vmc
+  LIST(APPEND C2_PP_MSDJ_SHORT "kinetic" "7.72378 0.012") # kinetic energy
+  LIST(APPEND C2_PP_MSDJ_SHORT "totenergy" "-11.059755 0.00099") # total energy
+  LIST(APPEND C2_PP_MSDJ_SHORT "eeenergy" "12.142072 0.0052") # e-e energy
+  LIST(APPEND C2_PP_MSDJ_SHORT "samples" "160000 0.0") # samples
+  QMC_RUN_AND_CHECK(vmc_short_pp_msdj-H5
                     "${CMAKE_SOURCE_DIR}/tests/molecules/C2_pp"
-                    C2_TZ_PP_MSDJ
-                    qmc_short_pp_msdj.xml
+                    C2_TZ_PP_SHORT_H5_MSDJ
+                    vmc_short_pp_msdj-H5.xml 
                     1 16
-                    ${MP_SUCCESS}
-                    1 C2MSDJS_VMC_SCALARS # VMC
+                    TRUE
+                    1 C2_PP_MSDJ_SHORT # VMC
                     )
-
-  QMC_RUN_AND_CHECK(short-C2_pp-msdj-traditional_vmc
+ 
+  QMC_RUN_AND_CHECK(vmc_short_pp_msdj-XML
```

Comment:
Please add back a test specific for the traditional algorithm code path. The default option is the table method.
You can find the input difference by comparing qmc_short_pp_msdj.xml and qmc_short_pp_msdj-traditional.xml in the develop branch.